src/main/java/uk/nhs/hee/tis/revalidation/connection/message/ConnectionMessage.java

```diff
@@ -32,4 +33,6 @@ public class ConnectionMessage {
   private String gmcId;
   private String designatedBodyCode;
   private LocalDate submissionDate;
+  // Use this field to tell us when gmc info last changed for this doctor.
```

Comment:
A Javadoc comment is preferable.  We aren't currently producing Javadocs but they are useful for IDE tooling too.